packages/widgets/src/Layer/Layer.js

```diff
@@ -54,13 +90,31 @@ const Layer = ({ children, className, level, position }) => {
 
 Layer.defaultProps = {
     position: 'fixed',
+    dataTest: 'dhis2-uicore-layer',
+    pointerEvents: 'all',
+    level: layers.applicationTop,
 }
 
+/**
+ * @typedef {Object} PropTypes
+ * @static
+ * @prop {string} [className]
+ * @prop {Node} [children]
+ * @prop {string} [dataTest=dhis2-uicore-layer]
+ * @prop {number} [level=layers.applicationTop]
+ * @prop {string} [pointerEvents=all] - One of 'all', 'none'
+ * @prop {boolean} [translucent] - When true a semi-transparent background is added
+ * @prop {function} [onClick]
+ */
 Layer.propTypes = {
     children: propTypes.node,
     className: propTypes.string,
+    dataTest: propTypes.string,
     level: propTypes.number,
-    position: propTypes.oneOf(['absolute', 'relative', 'fixed']),
+    pointerEvents: propTypes.oneOf(['all', 'none']),
```

Comment:
This enables "non-blocking" layers, which could theoretically be handy if you would want to visually "layer" elements on top of one another, but still allow interaction with the underlying layer. There'd be a gotcha here, which is that the layer content would need to make sure to set `pointer-events: all;` again.